public/newsitem.js

```diff
@@ -0,0 +1,32 @@
+function newsItem(apiResponseElement,articleID) {
+  this.headline = apiResponseElement.headline;
+  this.bodyText = apiResponseElement.bodyText;
+  this.shortUrl = apiResponseElement.shortUrl;
+  this.thumbnail = apiResponseElement.thumbnail;
+  this.id = articleID
+}
+
+newsItem.prototype.getHeadlineHTML= function() {
+  var headline = document.createElement('a')
+  headline.href = this.shortUrl;
+  headline.classList.add('headline');
+  headline.id = `Article #${this.id} headline`;
+  headline.textContent = this.headline;
+  return headline;
+};
```

Comment:
For HTML semantics you could have a h2 or h3 wrapped in the a? At the moment its semantically no different from the paragraph text below